spec/01-unit/08-entities_schemas_spec.lua

```diff
@@ -449,6 +452,230 @@ describe("Entities Schemas", function()
     end)
   end)
 
+  --
+  -- UPSTREAMS
+  --
+
+  describe("Upstreams", function()
+    local slots_default, slots_min, slots_max = 100, 10, 2^16
+
+    it("should require a valid `name` and no port", function()
+      local valid, errors, check
+      valid, errors = validate_entity({}, upstreams_schema)
+      assert.False(valid)
+      assert.equal("name is required", errors.name)
+
+      valid, errors, check = validate_entity({ name = "123.123.123.123" }, upstreams_schema)
+      assert.False(valid)
+      assert.Nil(errors)
+      assert.equal("Invalid name; no ip addresses allowed", check.message)
+
+      valid, errors, check = validate_entity({ name = "\\\\bad\\\\////name////" }, upstreams_schema)
+      assert.False(valid)
+      assert.Nil(errors)
+      assert.equal("Invalid name; must be a valid hostname", check.message)
+      
+      valid, errors, check = validate_entity({ name = "name:80" }, upstreams_schema)
+      assert.False(valid)
+      assert.Nil(errors)
+      assert.equal("Invalid name; no port allowed", check.message)
+      
+      valid, errors, check = validate_entity({ name = "valid.host.name" }, upstreams_schema)
+      assert.True(valid)
+      assert.Nil(errors)
+      assert.Nil(check)
```

Comment:
As previously mentioned multiple times before: we have always been using the `is_` form of luassert:
- `assert.is_table`
- `assert.is_string`
- `assert.is_true`
- `assert.is_nil`

Capital letters are confusing and carry less meaning than the `is_` prefix.